_conferences/womenwhocodemobilesummit.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "Women Who Code Mobile Summit 2022"
+website: https://www.womenwhocode.com/mobile/about
+location: Online
+online: true
+
+date_start: 2022-07-27
+date_end: 2022-07-27
+
+cfp:
+start:  2022-07-27
+end:    2022-07-27
+site:   https://hopin.com/events/women-who-code-mobile-summit/registration
```

Comment:
From the website it looks like speakers have already been selected and there is/was no public CfP. So you can leave out the whole `cfp` section.